### DISCUSSION TITLE
[Idea] RAG Tuning Of Assitants

### DISCUSSION THREAD

**Discussion by FlynnCruse** [lobehub/lobehub#666] [Ideas]
Open AI allows for people to RAG train, or fine-tune custom GPTs. 
Am I missing something that allows us to do that in Lobe-chat when creating assistants?

**Comment by arvinxx** [comment]
We haven't supported RAG when creating the assistant. And we have a plan to support this later.

More recently ,we may considering to add a OpenAI Assistant plugin.
